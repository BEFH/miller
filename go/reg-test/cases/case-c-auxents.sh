run_mlr lecat --mono < $indir/line-ending-cr.bin
run_mlr lecat --mono < $indir/line-ending-lf.bin
run_mlr lecat --mono < $indir/line-ending-crlf.bin

run_mlr_no_output termcvt --cr2lf   < $indir/line-ending-cr.bin   > $outdir/line-ending-temp-1.bin
run_mlr_no_output termcvt --cr2crlf < $indir/line-ending-cr.bin   > $outdir/line-ending-temp-2.bin
run_mlr_no_output termcvt --lf2cr   < $indir/line-ending-lf.bin   > $outdir/line-ending-temp-3.bin
run_mlr_no_output termcvt --lf2crlf < $indir/line-ending-lf.bin   > $outdir/line-ending-temp-4.bin
run_mlr_no_output termcvt --crlf2cr < $indir/line-ending-crlf.bin > $outdir/line-ending-temp-5.bin
run_mlr_no_output termcvt --crlf2lf < $indir/line-ending-crlf.bin > $outdir/line-ending-temp-6.bin

run_mlr hex < $outdir/line-ending-temp-1.bin
run_mlr hex < $outdir/line-ending-temp-2.bin
run_mlr hex < $outdir/line-ending-temp-3.bin
run_mlr hex < $outdir/line-ending-temp-4.bin
run_mlr hex < $outdir/line-ending-temp-5.bin
run_mlr hex < $outdir/line-ending-temp-6.bin
