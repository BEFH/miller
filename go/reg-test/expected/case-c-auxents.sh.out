
mlr lecat --mono
hello[CR]there[CR]how[CR]are[CR]you[CR]
mlr lecat --mono
hello[LF]
there[LF]
how[LF]
are[LF]
you[LF]

mlr lecat --mono
hello[CR][LF]
there[CR][LF]
how[CR][LF]
are[CR][LF]
you[CR][LF]

mlr termcvt --cr2lf

mlr termcvt --cr2crlf

mlr termcvt --lf2cr

mlr termcvt --lf2crlf

mlr termcvt --crlf2cr

mlr termcvt --crlf2lf

mlr hex
00000000: 68 65 6c 6c  6f 0a 74 68  65 72 65 0a  68 6f 77 0a |hello.there.how.|
00000010: 61 72 65 0a  79 6f 75 0a                           |are.you.|

mlr hex
00000000: 68 65 6c 6c  6f 0d 0a 74  68 65 72 65  0d 0a 68 6f |hello..there..ho|
00000010: 77 0d 0a 61  72 65 0d 0a  79 6f 75 0d  0a          |w..are..you..|

mlr hex
00000000: 68 65 6c 6c  6f 0d 74 68  65 72 65 0d  68 6f 77 0d |hello.there.how.|
00000010: 61 72 65 0d  79 6f 75 0d                           |are.you.|

mlr hex
00000000: 68 65 6c 6c  6f 0d 0a 74  68 65 72 65  0d 0a 68 6f |hello..there..ho|
00000010: 77 0d 0a 61  72 65 0d 0a  79 6f 75 0d  0a          |w..are..you..|

mlr hex
00000000: 68 65 6c 6c  6f 0d 74 68  65 72 65 0d  68 6f 77 0d |hello.there.how.|
00000010: 61 72 65 0d  79 6f 75 0d                           |are.you.|

mlr hex
00000000: 68 65 6c 6c  6f 0a 74 68  65 72 65 0a  68 6f 77 0a |hello.there.how.|
00000010: 61 72 65 0a  79 6f 75 0a                           |are.you.|

